containers/php/Dockerfile

```diff
@@ -98,10 +98,16 @@ RUN apt-get update && \
       cron \
     && apt-get autoremove \
     && find /var/log -type f -name "*.log" -delete \
-    && rm -rf /var/lib/apt/lists/* /var/cache/ldconfig/aux-cache
+    && rm -rf /var/lib/apt/lists/* /var/cache/ldconfig/aux-cache \
+    && rm -rf /etc/cron.*/*
+
+WORKDIR /var/www
+
+COPY cron/docker-cron-entrypoint /usr/local/bin/
 
-ENTRYPOINT ["crond"]
-CMD ["-f", "-l", "8"]
+HEALTHCHECK --interval=5s --timeout=3s CMD ps aux | grep '[c]ron' || exit 1
```

Comment:
Kannst du mir bitte erklären, was der Vorteil von diesem Health-Check ist? Wenn es diesen Prozess nicht mehr gibt, dann ist der Container doch ohnehin gestorben, weil der Cron ja PID 1 ist...

Reply:
Maybe this is a bit exaggerated in this context, but I can configure a dependency on the healthcheck:
```yaml
  cron:
    healthcheck:
      test: "ps aux | grep '[c]ron' || exit 1"
      interval: 5s
      timeout: 3s
      retries: 5
  fpm:
    depends_on:
      cron:
        condition: service_healthy
```
As it could take some time until the `entrypoint`-script exec the real command.